### ISSUE TITLE
[Feature Request]: Tags

### ISSUE THREAD

**Issue by bkmgit** [bkaradzic/GENie#573]
Please consider tagging commits corresponding to releases.

**Comment by bkaradzic** [comment]
Nah! You already have everything you need to find exact commit number:

`GENie - Project generator tool version 1181 (commit 29e6832fdf3b106c0906d288c8ced6c0761b8985)`

`1181` is `git rev-list --count HEAD`

or you can use sha1:

`29e6832fdf3b106c0906d288c8ced6c0761b8985`

**Comment by bkmgit** [comment]
Ok. Maybe it is helpful to automate this. As an example the current README states current release is:
```
version 1181 (commit 29e6832fdf3b106c0906d288c8ced6c0761b8985)
```
Yet the version information in is incorrect, see the [README](https://github.com/bkaradzic/GENie/blob/29e6832fdf3b106c0906d288c8ced6c0761b8985/README.md) and [version](https://github.com/bkaradzic/GENie/blob/29e6832fdf3b106c0906d288c8ced6c0761b8985/src/host/version.h) files. This is fixed in commit c4e45e25c16e66b1891123af72cd395729110be5 see the  [README](https://github.com/bkaradzic/GENie/blob/c4e45e25c16e66b1891123af72cd395729110be5/README.md) and [version](https://github.com/bkaradzic/GENie/blob/c4e45e25c16e66b1891123af72cd395729110be5/src/host/version.h) files. These files do not contain any
code changes, but it would be helpful to have the correct version number in the release commit.
